xla/service/gpu/tests/gpu_atomic_test.cc

```diff
@@ -100,24 +108,42 @@ TEST_F(GpuAtomicTest, TestAddAtomicF32) {
     }
 )";
 
-  CompileAndVerifyIr(hlo_string, is_built_with_rocm_ ? R"(
-CHECK: atomicrmw fadd ptr addrspace(1) %[[ADDR:.*]], float %[[VALUE:.*]] syncscope("agent") seq_cst
+  HloModuleConfig config;
+  DebugOptions debug_options = config.debug_options();
+  debug_options.set_xla_gpu_ftz(true);
+  config.set_debug_options(debug_options);
+
+  TF_ASSERT_OK_AND_ASSIGN(auto module,
+                          ParseAndReturnVerifiedModule(hlo_string, config));
+
+  CompileAndVerifyIr(std::move(module), is_built_with_rocm_ ? R"(
```

Comment:
Nit: I think this is a bit more readable as

```
if (is_build_with_rocm_) {
  CompileAndVerifyIr
} else {
  CompileAndVerifyIr
}
```

(Assuming there's some reason why the separate test suite doesn't work).